Source/WebCore/Modules/fetch/FetchRequestInit.h

```diff
@@ -47,6 +47,7 @@ struct FetchRequestInit {
     String integrity;
     std::optional<bool> keepalive;
     JSC::JSValue signal;
+    std::optional<FetchPriority> priority;
```

Comment:
optional or default value being `auto`?

Reply:
Maybe we can fine tune later, either way is fine to me.